tests/models/test_lora_layers.py

```diff
@@ -467,3 +454,98 @@ def test_lora_save_load_with_xformers(self):
 
         # Outputs shouldn't match.
         self.assertFalse(torch.allclose(torch.from_numpy(orig_image_slice), torch.from_numpy(lora_image_slice)))
+
+
+@slow
+@require_torch_gpu
+class LoraIntegrationTests(unittest.TestCase):
```

Comment:
Integration tests for the different lora cases, I cherry-picked these over to main and ran them there as well